google/logging/v2/logging_gapic.yaml

```diff
@@ -352,8 +352,11 @@ interfaces:
     bundling:
       thresholds:
         element_count_threshold: 100
-        request_byte_threshold: 1024 # 1 Kb
+        request_byte_threshold: 20480 # 20 Kb
```

Comment:
I think it's really just the flow control limit that really matters for this. The request_byte_threshold value just means that the bundle will be sent. 

Let's set `flow_control_byte_limit` to 4MB for now because of https://github.com/GoogleCloudPlatform/google-cloud-java/issues/1499 , and add a comment saying that it can be bumped up to 5MB later.